README.md

```diff
@@ -59,11 +59,31 @@ Even more data can be collected by adding the `--danger` flag:
 $ sudo mayday --danger
 ```
 
+## what's collected
+
+By default, mayday operates in a "safe" mode. No sensitive information is
+collected -- only information that support might need. This includes things like:
+
+* network connections, firewall rules, and hostname
+* information about currently running processes, including open files and ports
+* log files from running services
+* filesystem and memory usage information
+* information about docker and rkt containers, including network and state but NOT logs
+
+The following is collected only if the `--danger` flag is activated:
+
+* logs and environment variables of docker and rkt containers
+
+The following information is **never** collected:
```

Comment:
I'm a little hesitant to say "never", since if someone puts passwords in their logs...

(also: don't do that)